gradle-language-server/src/main/java/com/microsoft/gradle/GradleServices.java

```diff
@@ -83,7 +83,7 @@
 
 public class GradleServices implements TextDocumentService, WorkspaceService, LanguageClientAware {
 
-  public static final List<String> supportedCommands = List.of("gradle.getDependencies");
+  public static final List<String> supportedCommands = List.of("gradle.getDependencies", "gradle.wrapperChanged");
```

Comment:
Use a more generic name for the new command, such as `"gradle.distributionChanged"`?

Reply:
This command will be triggered only when the `gradle-wrapper.properties` changes. If the related setting changes, we use `didChangeConfiguration` to handle.